docs/querying/sql-functions.md

```diff
@@ -807,28 +807,85 @@ Returns the cumulative distribution of the current row within the window calcula
 
 ## CURRENT_DATE
 
-`CURRENT_DATE`
+Returns the current date in the connection's time zone.
 
-**Function type:** [Scalar, date and time](sql-scalar.md#date-and-time-functions)
+* **Syntax:** `CURRENT_DATE`
+* **Function type:** Scalar, date and time
 
-Returns the current date in the connection's time zone.
+<details><summary>Example</summary>
 
-## CURRENT_TIMESTAMP
+The following example returns the current date.
 
-`CURRENT_TIMESTAMP`
+```sql
+SELECT CURRENT_DATE AS "current_date"
+```
 
-**Function type:** [Scalar, date and time](sql-scalar.md#date-and-time-functions)
+Returns the following:
+
+| `current_date` | 
+| -- |
+| `2024-08-14T00:00:00.000Z `|
+
+</details>
+
+[Learn more](sql-scalar.md#date-and-time-functions)
+
+## CURRENT_TIMESTAMP
 
 Returns the current timestamp in the connection's time zone.
```

Comment:
This is returning UTC for me. For example it is 15:16 right now locally and I get: `2024-08-21T22:16:59.342Z`